scripts/update_ci.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+
+set -eu
+
+if [ ! -f "template_config.yml" ]; then
+  echo "No template_config.yml detected."
+  exit 1
+fi
+
+plugin_name = python ../plugin_template/scripts/get_template_config_value.py plugin_name
+ci_update_docs = python ../plugin_template/scripts/get_template_config_value.py ci_update_docs
+noissue_marker = python ../plugin_template/scripts/get_template_config_value.py noissue_marker
```

Comment:
Are you sure this works in bash? I thought i'd be:
```suggestion
plugin_name="$(python ../plugin_template/scripts/get_template_config_value.py plugin_name)"
ci_update_docs="$(python ../plugin_template/scripts/get_template_config_value.py ci_update_docs)"
noissue_marker="$(python ../plugin_template/scripts/get_template_config_value.py noissue_marker)"
```